pkgs/os-specific/bsd/freebsd/patches/14.1/sys-cdefs-static-assert.patch

```diff
@@ -0,0 +1,41 @@
+From 22cdafe197ac960c5ce839ef6ec699b59f4b0080 Mon Sep 17 00:00:00 2001
```

Comment:
Where is this patch used?

Reply:
The entire patches directory is scanned and patches (or hunks thereof!) are automatically included based on their relevance to the files that are included in a given derivation. It's like we're always patching the entire tree, but we don't mass rebuild any time any patch changes.

References: [1](https://github.com/NixOS/nixpkgs/blob/7f3288978b0b4fa765b027903ed2adf6212d946b/pkgs/os-specific/bsd/freebsd/default.nix#L49) [2](https://github.com/NixOS/nixpkgs/blob/7f3288978b0b4fa765b027903ed2adf6212d946b/pkgs/os-specific/bsd/freebsd/pkgs/mkDerivation.nix#L116-L121) [3](https://github.com/NixOS/nixpkgs/blob/7f3288978b0b4fa765b027903ed2adf6212d946b/pkgs/os-specific/bsd/freebsd/lib/default.nix#L67)